universal-login-example/src/services/StorageService.js

```diff
@@ -0,0 +1,72 @@
+import localforage from 'localforage';
+
+const IDENTITY_KEY = 'universal-login-identity';
+const DEVICES_KEY = 'universal-login-devices';
+
+class StorageService {
+  checkForIdentity() {
+    return localforage.getItem(IDENTITY_KEY).then(function (value) {
+      return value;
+    }).catch(function() {
+      return false;
+    });
+  }
+
+  storeIdentity(identity) {
+    return localforage.setItem(IDENTITY_KEY, identity).then(function () {
+      return true;
+    }).catch(function() {
+      return false;
+    });
+  }
+
+  getDevices() {
+    return localforage.getItem(DEVICES_KEY).then(function (value) {
+      return value;
+    }).catch(function() {
+      return false;
+    });
+  }
+
+  async clearStorage() {
+    localforage.clear();
```

Comment:
Does clear returns promise? if so it should be:
`return localforage.clear();`
which in this case is the same as:
`await localforage.clear();`